test/files/run/lazylist-dropwhile-stack-safe.scala

```diff
@@ -1,10 +0,0 @@
-object Test extends App {
-  // Should not overflow the stack
-  val lazyList = Iterator.iterate(LazyList.from(0))(_.dropWhile(_ => false)).drop(10000).next
-  lazyList.head
-  lazyList.tail.head
-
-  val stream = Iterator.iterate(Stream.from(0))(_.dropWhile(_ => false)).drop(10000).next
-  stream.head // superfluous, because `head` should be eager
-  stream.tail.head
-}
```

Comment:
Why did you remove that test?

Reply:
The test was added in #6608, which made `drop` and `dropWhile` stack-safe but not lazy. Since I changed them back to being lazy, they are no longer stack-safe